.github/workflows/validate-cg-manifest.sh

```diff
@@ -125,10 +125,11 @@ do
     else
       # Try a few times to download the source listed in the manifest
       mkdir -p ./cgmanifest_test_dir
-      for _ in {1..10}
+      for attempt in {1..10}
       do
         wget --quiet -P ./cgmanifest_test_dir "$manifesturl" && touch ./cgmanifest_test_dir/WORKED && break
```

Comment:
Doesn't need to be in this PR, but we may want to consider using the `--spider` option with `wget` to check for a URL's existence without downloading the file. Could save us a decent amount of time.

Reply:
Thanks for the hint! I'm actually hoping to do some improvements to this script as part of the `cgmanifest.json` clean-up work and making the checks more efficient is one of them, so this should fit nicely into that.:)